packages/mobile/src/analytics/Properties.tsx

```diff
@@ -273,6 +273,10 @@ interface VerificationEventsProperties {
     issuer: any
     error: string
   }
+  [VerificationEvents.verification_reveal_attestation_status]: {
+    issuer: any
+    status: any
```

Comment:
I see you just pass forward the entire json response from the attestation service to this event.  Does the event thrown include all fields even though only these two are defined in the Properties.tsx?

I'm not super familiar with this part of the metrics pipeline and how segment may use this file, but I wonder if we need to explicitly include each property here for segment to include the additional fields... I would think @annakaz @tarikbellamine know